code/game/objects/items/weapons/melee/swords.dm

```diff
@@ -755,7 +755,7 @@
 //................ Zizo Sword ............... //
 /obj/item/weapon/sword/long/greatsword/zizo
 	name = "darksteel kriegsmesser"
-	desc = "A dark red curved blade. Called forth from Her will, if you wield this blade you are to be feared, if you do not, you are dead."
+	desc = "A dark red curved blade. Called forth from Her will, if you wield this blade you are to be feared, if you do not, you are dead.<br><span class='warningbig'>ZIZOID HERESY!!</span>"
```

Comment:
add a patron check on these . if you are ZIZOID , it should be a green text saying something good instead

Reply:
also .. probably you should add the ZIZOID heresy thing on "examine" proc rather than directly on the description